lib/registration.pm

```diff
@@ -231,15 +231,15 @@ sub select_addons_in_textmode {
     if ($flag) {
         send_key_until_needlematch 'scc-module-area-selected', 'tab';
         send_key_until_needlematch "scc-module-$addon",        'down';
-        send_key 'spc';
+        wait_screen_change { send_key 'spc' };
```

Comment:
But we are doing the "wait_still_screen" just afterwards. That does not make sense to me